src/components/Post.jsx

```diff
@@ -75,8 +76,19 @@ function Post(prop) {
     setComment("");
   };
 
+  const deleteComment = async (event, Commentref) => {
+    console.log(Commentref.content.text);
+    event.preventDefault();
+    db
```

Comment:
async function should've an await expression, line 82 could be like await db.

Reply:
sorry forgot it